corehq/apps/export/views.py

```diff
@@ -488,8 +488,11 @@ def download_export_form(self):
         raise NotImplementedError("You must implement download_export_form.")
 
     @staticmethod
-    def get_export_schema(export_id):
-        return get_document_or_404_lite(SavedExportSchema, export_id)
+    def get_export_schema(domain, export_id):
+        doc = get_document_or_404_lite(SavedExportSchema, export_id)
```

Comment:
ah right i just took a closer look at that function and realizd why it wouldn't work